Gemfile

```diff
@@ -4,6 +4,7 @@ source 'https://rubygems.org'
 gem 'rails', '4.1.7'
 
 gem 'mysql2'
+gem 'pg'
```

Comment:
After reading a bit about the topic, I'd suggest to define groups for the two gems.

```ruby
gem 'mysql2', group: 'mysql'
gem 'pg', group: 'postgresql'
```

This will allow people to ignore the database provider they don't need by doing `bundle install --without mysql`.